crystlib.py

```diff
@@ -0,0 +1,105 @@
+"""
+Module for getting crystals info from input indexin stream file.
+"""
+
+from stream_read import diction_crystal_return
+# From the stream file return a dictionary with each crystal as key and
+# crystal details as parameters.
+
+
+class Crystal():
```

Comment:
parenthesis are not necessary

Reply:
Addressed in: 59fa0f7805e13bcd3c89c653d35783ae3cab01bb ea1b915ab97c9520c17c47a22ac785a9dfec6916